src/ClientHandle.cpp

```diff
@@ -197,6 +195,8 @@ void cClientHandle::Destroy(void)
 		}
 	}
 	player->RemoveClientHandle();
+	SetState(csDestroyed);  // Tick thread is allowed to call destructor async at any time after this
+	m_Self.reset();
```

Comment:
Race condition, tick thread could delete the client while this function is still running, so accessing `m_PlayerPtr` was invalid.